set-version.sh

```diff
@@ -2,6 +2,13 @@
 
 NEW_VERSION=$1
 
+# Convert NPM version to semver compatible if needed
+if [[ $NEW_VERSION =~ [0-9]+.[0-9]+.[0-9]+.[a-z] ]]; then
+  NEW_NPM_VERSION=$(echo $NEW_VERSION | awk -F '.' '{ print $1"."$2"."$3 }').$(echo $NEW_VERSION | cut -d '.' -f 4 | sed 's/-/+/g')
```

Comment:
Tried to avoid that since it would convert `22.0.0-alpha` to `22.0.0+alpha` in case we ever wanted to do a pre-release, but we can fix that if we ever do it (which is unlikely tbh).

Reply:
Ahh yes, I overlooked this. Looks like this has now been changed to `22.0.3+redhat-00001` so that should be valid.